packages/fscomponents/src/components/MultiCarousel/MultiCarousel.web.tsx

```diff
@@ -281,9 +281,13 @@ export const MultiCarousel = <ItemT, >(props: MultiCarouselProps<ItemT>) => {
   );
 
   const goToNext = useCallback(async (options?: GoToOptions | GestureResponderEvent) => {
-    const nextIndex = currentIndex + 1 > numberOfPages - 1 ? numberOfPages - 1 : currentIndex + 1;
+    const nextIndex = currentIndex + 1 > numberOfPages - 1 ? 0 : currentIndex + 1;
+    const animationOptions =
+      nextIndex !== 0 &&
```

Comment:
Other functions in this component use this function to advance the pages. Here you are always turning off animation if the nextIndex is 0. This means even without autoplay, if a user wants to use animation for the arrows, it will never work. It appears very broken. You would need to remove this logic and handle this in the useEffect or create another function that the useEffect calls. In any case, from a UX perspective...having transitions and those transitions suddenly drop out does not seem intentional and can be very jolting to some users.